content/concepts/did-ddo.md

```diff
@@ -1,173 +1,607 @@
+| **`address`** | `string`               | Contract address of the deployed ERC721 NFT contract.                     |
+| **`name`**    | `string`               | Name of NFT set in contract.                                              |
+| **`symbol`**  | `string`               | Symbol of NFT set in contract.                                            |
+| **`owner`**   | `string`               | ETH account address of the NFT owner.                                     |
+| **`state`**   | `number`               | State of the asset reflecting the NFT contract value. See [State](#state) |
+| **`created`** | `ISO date/time string` | Contains the date of NFT creation.                                        |
+
+Example:
+
+```json
+{
+  "nft": {
+    "adddress": "0x000000",
+    "name": "Ocean Protocol Asset v4",
+    "symbol": "OCEAN-A-v4",
+    "owner": "0x0000000",
+    "state": 0,
+    "created": "2000-10-31T01:30:00Z"
+  }
+}
+```
+
+### Datatokens
+
+The `datatokens` array contains information about the ERC20 datatokens attached to [asset services](#services).
+
+| Attribute       | Type     | Description                                      |
+| --------------- | -------- | ------------------------------------------------ |
+| **`address`**   | `string` | Contract address of the deployed ERC20 contract. |
+| **`name`**      | `string` | Name of NFT set in contract.                     |
+| **`symbol`**    | `string` | Symbol of NFT set in contract.                   |
+| **`serviceId`** | `string` | ID of the service the datatoken is attached to.  |
+
+Example:
+
+```json
+{
+  "datatokens": [
+    {
+      "adddress": "0x000000",
+      "name": "Datatoken 1",
+      "symbol": "DT-1",
+      "serviceId": "1"
+    },
+    {
+      "adddress": "0x000001",
+      "name": "Datatoken 2",
+      "symbol": "DT-2",
+      "serviceId": "2"
+    }
+  ]
+}
+```
+
+### Event
+
+The `event` section contains information about the last transaction that created or updated the DDO.
+
+Example:
 
 ```json
 {
-  "type": "credential3Box",
-  "values": ["profile1", "profile2"]
+  "event": {
+    "tx": "0x8d127de58509be5dfac600792ad24cc9164921571d168bff2f123c7f1cb4b11c",
+    "block": 12831214,
+    "from": "0xAcca11dbeD4F863Bb3bC2336D3CE5BAC52aa1f83",
+    "contract": "0x1a4b70d8c9DcA47cD6D0Fb3c52BB8634CA1C0Fdf",
+    "datetime": "2000-10-31T01:30:00"
+  }
+}
+```
+
+### Statistics
+
+The `stats` section contains different statistics fields.
+
+| Attribute      | Type     | Description                                                                                                   |
+| -------------- | -------- | ------------------------------------------------------------------------------------------------------------- |
+| **`consumes`** | `number` | How often an asset was consumed, meaning how often it was either downloaded or used as part of a compute job. |
+| **`isInPurgatory`** | `string`  | If asset is listed in purgatory and reason |
```

Comment:
first of all, this "Statistics" section just became a random section by adding in `isInPurgatory` so I would suggest also stating that by calling it "Miscellaneous/`misc`" or something.

But you stated "If asset is listed in purgatory and reason", yet the example only shows "true/false" so might make sense giving purgatory its own section like:

```
"purgatory": {
  "state": true
  "reason": "Copyright violation"
}
```

Reply:
agree with suggested format